core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1549,6 +1549,22 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_FLUSH_SIZE_MAX =
+      new Builder(Name.MASTER_EMBEDDED_JOURNAL_FLUSH_SIZE_MAX)
+          .setDefaultValue("64MB")
+          .setDescription("The maximum size in bytes of journal entries allowed "
+              + "in concurrent journal flushing (journal IO to standby masters "
+              + "and IO to local disks.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.MASTER)
+          .build();
+  public static final PropertyKey MASTER_EMBEDDED_JOURNAL_JOURNAL_SIZE_MAX =
+      new Builder(Name.MASTER_EMBEDDED_JOURNAL_JOURNAL_SIZE_MAX)
+          .setDefaultValue("4MB")
```

Comment:
Should/can this be larger? Is there any downside for a larger max size?

Reply:
That's the default value in Ratis. If we want to increase this value, we may hit other thresholds.
For example, FLUSH_SIZE_MAX property should also be increased. ENTRY_SIZE_MAX defines the single journal entry allowed, FLUSH_SIZE_MAX one define the total concurrent journal entry size allowed. Other thresholds are unknown. But i agree that we probably should enlarge the default value of these two properties together

Reply:
Enlarge the value of both properties, it should not have downside in normal cases. If we figure out any downside, i would move the value back